schemas/src/main/java/com/scottlogic/deg/schemas/v3/RuleDeserializer.java

```diff
@@ -1,8 +1,10 @@
 package com.scottlogic.deg.schemas.v3;
```

Comment:
Will this change to schemas code affect the Profiler repository?

Reply:
Potentially, however it I expect it creates constraints within the scope of a rule, the removed functionality only allows for expediency for people to write profiles really.